source/jormungandr/jormungandr/scenarios/tests/journey_compare_tests.py

```diff
@@ -948,3 +952,166 @@ def test_heavy_journey_bss():
 
     assert 'to_delete' not in journey.tags
 
+
+def test_activate_deactivate_mini_bike():
+    """
+
+      A                 B                           C            D
+      *................*---------------------------*.............*
+      A: origin
+      D: Destination
+      A->B : Bike
+      B->C : public transport
+      C->D : Bike
+
+    """
+    # case 1: request without origin_mode and destination_mode
+    request = {'_min_bike': 10}
+
+    journey = response_pb2.Journey()
+    journey.sections.add()
+    journey.sections[-1].type = response_pb2.STREET_NETWORK
+    journey.sections[-1].street_network.mode = response_pb2.Bike
+    journey.sections[-1].duration = 5
+
+    journey.sections.add()
+    journey.sections[-1].type = response_pb2.PUBLIC_TRANSPORT
+    journey.sections[-1].street_network.mode = response_pb2.PUBLIC_TRANSPORT
+    journey.sections[-1].duration = 35
+
+    journey.sections.add()
+    journey.sections[-1].type = response_pb2.STREET_NETWORK
+    journey.sections[-1].street_network.mode = response_pb2.Bike
+    journey.sections[-1].duration = 7
+
+    journey_filter._filter_too_short_heavy_journeys([journey], request)
+    assert 'to_delete' not in journey.tags
+
+    # case 2: request without origin_mode
+    journey.sections[-1].duration = 15
```

Comment:
minor: Would be easier to read and understand if you only change `_min_bike` value (same for all tests almost - maybe create 2 journeys)